src/tod_manager.hpp

```diff
@@ -230,13 +236,17 @@ class tod_manager
 		std::vector<time_of_day> times_;
 		std::vector<area_time_of_day> areas_;
 
+		//max liminal bonus
+		int liminal_bonus_;
```

Comment:
I'm not clear on the meaning of this. The expression `max liminal bonus` seems to imply that it's some limit on the liminal bonus, but the code itself seems to suggest that it's actually the exact liminal bonus for this specific time.

Reply:
tod_manager is not a specific time, but a global (scenario) object that knows about the schedule.

Reply:
Oh, I guess I got confused about where its defined. Okay then.